src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.DeclarationInfo.cs

```diff
@@ -392,6 +427,17 @@ private static Accessibility GetAccessibility(SyntaxTokenList modifiers)
 
                 return Accessibility.NotApplicable;
             }
+
+            private static SyntaxNode GetTypeSyntaxOfArgument(ArgumentSyntax argumentSyntax)
+            {
+                switch (argumentSyntax.Expression?.Kind())
+                {
+                    case SyntaxKind.DeclarationExpression:
+                        return (argumentSyntax.Expression as DeclarationExpressionSyntax).Type;
+                    default:
+                        return argumentSyntax.Expression;
```

Comment:
This seems odd... you asked for the type syntax, but you're returning just hte expression?  I can envision why you want this.  But this line def needs a comment explaining why this is ok.